app/Http/Controllers/Apis/Protected/Summit/OAuth2SummitAttendeesApiController.php

```diff
@@ -446,6 +448,8 @@ function () {
                     'summit_hall_checked_in_date' => 'sometimes|date_format:U',
                     'tags' => 'sometimes|string',
                     'tags_id' => 'sometimes|integer',
+                    'notes' => 'sometimes|string',
+                    'has_notes' => 'sometimes|required|string|in:true,false',
```

Comment:
this is not correct use Boolean Mapping

  'has_notes' => ['sometimes', new Boolean()]